docs/synapse/userguides/ug017_storm_ref_stats.rst

```diff
@@ -0,0 +1,16 @@
+Storm Reference - Statistical Operators
+=======================================
+
+Todo
+
+See the `Storm Syntax Conventions`__ for an explanation of the usage format used below.
+
+Where specific query examples are given, they are commonly provided in pairs using operator syntax followed by the equivalent macro syntax (if available).
+
+stat()
+------
+Todo
+
+
+.. _conventions: ../userguides/ug011_storm_basics.html#syntax-conventions
+__ conventions_
```

Comment:
This page looks good to me as long as the PRs will be done in another PR (and weren't intended to be done in this one)